tests/integration/test_api_gateway.py

```diff
@@ -29,6 +29,7 @@
 # endpoint paths
 API_PATH_DATA_INBOUND = '/data'
 API_PATH_HTTP_BACKEND = '/hello_world'
+# API_PATH_LAMBDA_PROXY_BACKEND = 'arn:aws:apigateway:hello_world:lambda:path'
```

Comment:
I would assume that this will become something like:
```
API_PATH_LAMBDA_PROXY_BACKEND = '/hello_world_from_lambda'
```
... and then we can pass in that variable as the `path` parameter to the `connect_api_gateway_to_http_with_lambda_proxy` function further below.